### DISCUSSION TITLE
Python Version

### DISCUSSION THREAD

**Discussion by warengonzaga** [WarenGonzaga/wifi-passview#20] [Brainstorm]
The python version would be a re-write of the original batch script.
I might improve the interface as well as the experience of using it.

Here's the possible support.
- Windows (of course)
- Mac
- Linux

I might add support with wireless adapters out there.
Everything here is possible I just need time to make it happen also [your support](https://buymeacoffee.com/warengonzaga).

Also, visit the official [Open Collective](https://opencollective.com/wifi-passview) page for this page.

**Comment by dldygnl** [comment]
i have created Windows and Linux Version

**Reply by warengonzaga** [reply]
Please use the dev branch for the development.

**Reply by warengonzaga** [reply]
@itsjeffersonli you can use this https://github.com/yorevs/hspylib

**Comment by warengonzaga** [comment]
Any update on this @itsjeffersonli? What if instead of creating a python version of it why not start an experimental version in Nim language?
